src/egithub_webhook.erl

```diff
@@ -25,13 +25,15 @@
                     }.
 -type file() :: map().
 -type req_data() :: map().
--export_type([req_data/0, message/0, file/0]).
+-type webhook_target_url() :: string()|undefined.
+-export_type([req_data/0, message/0, file/0, webhook_target_url/0]).
 
 -callback handle_pull_request(egithub:credentials(), req_data(), [file()]) ->
   {ok, [message()]} | {error, term()}.
 
 -callback handle_error({error, term()}, req_data(), [file()]) ->
-  {error, {failed, integer()}, string()} | {ok, [map()], string()}.
+    {error, term(),  webhook_target_url()}
```

Comment:
@elbrujohalcon don't know if we want to make this spec more generic, because we had the error tuple `{error, {failed, integer()}, string()}`, and now @mfelsche suggest `{error, term(),  webhook_target_url()}`.